pressbooks/pressbooks-book-directory-fe
# Setup universal analytics

This PR solves #412 

This PR adds the UA in addition to the existing GA4.

#### How to test

1. Download and test the app everything should work as expected.
1. Run the build process to see if the dist/index.html is generated according to the required settings
Create a `VITE_APP_GUA_MEASUREMENT_ID` in your .env and review that value is added to the dist/index.html after running npm run build
